### DISCUSSION TITLE
【灌水】

### DISCUSSION THREAD

**Discussion by blacktree-eme** [share121/inter-knot#2815] [其他]
![屏幕截图(181)](https://github.com/user-attachments/assets/59f15ac3-687b-44fe-8f99-956da0c4bfbd)
![屏幕截图(117)](https://github.com/user-attachments/assets/c91883f9-1e78-474c-b10c-6b86c37ee904)
这种情况的概率是多少？

**Comment by 1luik** [comment]
十连三金
